doc/API.md

```diff
@@ -1,161 +1,8 @@
-
 # Torque API
 
+Torque.js is an efficient and stylish rendering method to animate your data. Torque.js uses TileCubes, which are JSON representations of multidimensional data with geospatial coordinates, to render data on the client.
 
-## L.TorqueLayer(options)
-
-One of two core classes for the Torque library - it is used to create an animated torque layer with custom settings.
-
-### Usage example
-
-```js
-  // initialize a torque layer that uses the CartoDB account details and SQL API to pull in data
-  var torqueLayer = new L.TorqueLayer({
-    user       : 'viz2',
-    table      : 'ow',
-    cartocss:  CARTOCSS
-  });
-```
-
-### Options
-
-##### Provider options
-
-| Option    | type       | Default   | Description                            | Options |
-|-----------|:-----------|:----------|:---------------------------------------|---------|
-| provider  | string     | ```sql_api```   | Where is the data coming from    | `sql_api`, `url_template`, or `windshaft` |
-
-##### CartoDB data options (SQL API provider)
```

Comment:
In general I don't see the point of separating functions and parameters according to a thin taxonomy such as data/style/etc, especially considering that there aren't that many of them.